user-service/README.md

```diff
@@ -1,26 +1,104 @@
+<!---
+Copyright 2020 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# <img src="../logos/logo.svg" width="180">
+
 # User Service
 
-<span style="color:red">**Note:** As noted in the [documentation root](../README.md) Palisade is in an early stage of development, therefore the precise APIs contained in these service modules are likely to adapt as the project matures.</span>
+The User Service is responsible for providing the other Palisade components with knowledge of the users of the system. 
+Note that we are avoiding having a separate notion of "Palisade" users that are distinct from the normal user accounts of the system. There is no such thing as a "Palisade" user.  
+One of the purposes of this service is to allow Palisade to adopt whatever notion of *user* the host environment has. For example, this may from a central directory service such as LDAP, 
```

Comment:
```suggestion
One of the purposes of this service is to allow Palisade to adopt whatever notion of *user* the host environment has. For example, this may come from a central directory service such as LDAP, 
```

Reply:
Done